documentation/leo/03_language.md

```diff
@@ -524,6 +522,8 @@ a = a.add(1u8);
 
 ### Arithmetic Operators
 
+Note: The right hand side of shift and exponentiation operators must be of type `u8`, `u16` or `u32`.
```

Comment:
```suggestion
```

This note is documented in the reference for Leo operators https://developer.aleo.org/leo/operators#shl

Could you add direct links for each of the operators in the table?

- [ ] shl - https://developer.aleo.org/leo/operators#shl
- [ ] shl_wrapped - https://developer.aleo.org/leo/operators#shl_wrapped
- [ ] shr - https://developer.aleo.org/leo/operators#shr
- [ ] shr_wrapped - https://developer.aleo.org/leo/operators#shr_wrapped